packages/components/src/item-group/styles.ts

```diff
@@ -33,6 +33,7 @@ export const itemWrapper = css`
 `;
 
 export const item = css`
+	box-sizing: border-box;
```

Comment:
This change was necessary to fix an issue with the separators 

<img width="903" alt="Screenshot 2023-01-06 at 16 44 40" src="https://user-images.githubusercontent.com/1083581/211046575-72565fd3-51aa-4cbf-a9fe-33a65700cd6b.png">

Reply:
Linking to #42415 for posterity.